packages/react/script/prop-docs/build.ts

```diff
@@ -0,0 +1,149 @@
+const docgen = require('react-docgen-typescript')
```

Comment:
Can we use module `import`s here instead of `require`? I see we're using `tsx` so that should be possible?